### PR TITLE
Update instances of `Optimint` to `Rollmint`

### PR DESCRIPTION
**Pull request by jcstein**
Also add new image and update

### CODE REVIEW

File: docs/developers/cosmwasm-dependency.md [hunk 1]
```diff
@@ -54,15 +54,15 @@ Find the right instructions specific for your OS.
...
+with Rollmint. Rollmint is a drop-in replacement for Tendermint that allows
 Cosmos-SDK applications to connect to Celestia's Data Availability network.
 
 ```sh
 git clone https://github.com/CosmWasm/wasmd.git
 cd wasmd
 git fetch --tags
 git checkout v0.27.0
-go mod edit -replace github.com/cosmos/cosmos-sdk=github.com/celestiaorg/cosmos-sdk@v0.45.4-optimint-v0.3.5
+go mod edit -replace github.com/cosmos/cosmos-sdk=github.com/celestiaorg/cosmos-sdk@v0.45.4-rollmint-v0.3.5
```
Comment (tzdybal) [inline docs/developers/cosmwasm-dependency.md:65]:
```suggestion
go mod edit -replace github.com/cosmos/cosmos-sdk=github.com/celestiaorg/cosmos-sdk-rollmint@v0.46.1-rollmint-v0.4.0
```

File: docs/developers/install-optimint.md [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
+Rollmint on our codebase.
 
-## Installing Optimint
+## Installing Rollmint
 
 Run the following command inside the `wordle` directory.
 
 ```sh
-go mod edit -replace github.com/cosmos/cosmos-sdk=github.com/celestiaorg/cosmos-sdk@v0.45.4-optimint-v0.3.5
+go mod edit -replace github.com/cosmos/cosmos-sdk=github.com/celestiaorg/cosmos-sdk@v0.45.4-rollmint-v0.3.5
```
Comment (tzdybal) [inline docs/developers/install-optimint.md:15]:
```suggestion
go mod edit -replace github.com/cosmos/cosmos-sdk=github.com/celestiaorg/cosmos-sdk-rollmint@v0.46.1-rollmint-v0.4.0
```

**Review by tzdybal — CHANGES_REQUESTED** [review]
You need to rename `install-optimint.md` to `install-rollmint.md`, because link is already updated.
Probably, you should also rename `optimint.md` to `rollmint.md` and fix links (if any). It seems that translation files also contains references to those files.

I also suggested the correct version of rollmint in comments.

**Comment by jcstein** [comment]
changes added, thanks again @tzdybal

**Review by tzdybal — APPROVED** [review]
LGTM :)

**Pull request merged**